entities/console.rb

```diff
@@ -0,0 +1,131 @@
+module Console
+  include Validation
+
+  def start
+    puts I18n.t(:greeting)
+    puts I18n.t(:exit)
+
+    gets.chomp
+  end
+
+  def name_input
+    loop do
```

Comment:
Could you please use recursion instead of `loop`?

Also, could you mark the comment as resolved only if you have made changes by this comment or answered it?